### DISCUSSION TITLE
getting error max fee per gas less than base fee in useSendTransaction

### DISCUSSION THREAD

**Discussion by ahmadkhan345** [wevm/wagmi#2409] [Q&A]
This is my code: 
 const customGasPrice = 50;

  const { data, sendTransaction } = useSendTransaction({
    request: {
      from: address,
      to: String(txDetails.to),
      data: String(txDetails.data),
      value: String(txDetails.value),
      gasPrice:customGasPrice
    },
  })
 and the ereror that I get is :
code: -32000
message : "err: max fee per gas less than block base fee: address 0xe4f1871A5583b6160AB3f8788BbeeDC80B2Ac5a4, maxFeePerGas: 50 baseFee: 32854624775 (supplied gas 15010499)"

**Comment by jxom** [comment]
what are you passing as the `gasPrice`? A minimal reproduction via https://new.wagmi.sh would be helpful.

**Reply by ahmadkhan345** [reply]
I am passing 50 for the moment

**Reply by jxom** [reply]
You’re probably after `parseGwei(‘50’)`

**Reply by ahmadkhan345** [reply]
I tried that, but it threw an error that parseGwei is not defined

**Reply by jxom** [reply]
You’ll need to import it from `viem`.

**Reply by ahmadkhan345** [reply]
Thank you for the replies.

**Reply by ahmadkhan345** [reply]
Its working now, so thank you.
